Code/Uplynk/Source/UplynkSSAIConfiguration.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  UplynkServerSideAdIntegrationConfiguration.swift
+//
+//  Created by Raffi on 09/09/2024.
+//  Copyright © 2024 THEOplayer. All rights reserved.
+//
+
+import THEOplayerSDK
+
+public class UplynkServerSideAdIntegrationConfiguration: THEOplayerSDK.CustomServerSideAdInsertionConfiguration {
```

Comment:
Trivial: Namespacing `THEOplayerSDK.` can be removed?

Reply:
Also the type name can also be changed to match the file name `UplynkSSAIURLBuilder` ?